addons/l10n_ro_edi/tests/test_xml_ubl_ro.py

```diff
@@ -0,0 +1,162 @@
+from odoo import Command
+from odoo.addons.l10n_account_edi_ubl_cii_tests.tests.common import TestUBLCommon
+from odoo.addons.account.tests.test_account_move_send import TestAccountMoveSendCommon
+from odoo.exceptions import UserError
+from odoo.tests import tagged
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestUBLRO(TestUBLCommon, TestAccountMoveSendCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref="ro"):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+        cls.company_data['company'].write({
+            'country_id': cls.env.ref('base.ro').id,
+            'state_id': cls.env.ref('base.RO_B').id,
+            'currency_id': cls.env.ref('base.RON').id,
+            'name': 'Hudson Construction',
+            'city': 'SECTOR1',
+            'zip': '010101',
+            'vat': 'RO1234567897',
+            'phone': '+40 123 456 789',
+            'street': "Strada Kunst, 3",
+            'invoice_is_ubl_cii': True,
+        })
+
+        cls.env['res.partner.bank'].create({
+            'acc_type': 'iban',
+            'partner_id': cls.company_data['company'].partner_id.id,
+            'acc_number': 'RO98RNCB1234567890123456',
+        })
+
+        cls.partner_a = cls.env['res.partner'].create({
+            'country_id': cls.env.ref('base.ro').id,
+            'state_id': cls.env.ref('base.RO_B').id,
+            'name': 'Roasted Romanian Roller',
+            'city': 'SECTOR3',
+            'zip': '010101',
+            'vat': 'RO1234567897',
+            'phone': '+40 123 456 780',
+            'street': "Rolling Roast, 88",
+            'ref': 'ref_partner_a',
+        })
+
+        cls.tax_19 = cls.env['account.tax'].create({
+            'name': 'tax_19',
+            'amount_type': 'percent',
+            'amount': 19,
+            'type_tax_use': 'sale',
+            'country_id': cls.env.ref('base.ro').id,
+        })
+
+    ####################################################
+    # Test export - import
+    ####################################################
+
+    def create_post_send_invoice(self, move_type):
```

Comment:
I would rather say create_post_send_move since you don't use it only for invoices